data_structures/stack_using_queue.cpp

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file
+ * @brief Implement a last in first out (LIFO) stack using queues.
+          The implemented stack should support all the functions of a normal queue (push, top, pop, and empty).
+ * @author [Neha Labhasetwar](https://github.com/nehalabhasetwar)
+ */
+
+#include <iostream>
+#include <queue>
+#include <cassert>
+
+class MyStack
+{
+    /** initialize queue */
+    std::queue <int> q;
```

Comment:
Add a brief description of the variable here (in the code).
```suggestion
    std::queue <int> q;  ///> description
```